packages/template/src/dev-tool/dev-tool-core.ts

```
@@ -1754,6 +1781,1358 @@ function createAITab(app: StackClientApp<true>): HTMLElement {
+        return;
+      }
+      serverHeatmap = { path: currentPath, totalClicks: 0, selectors: [], elements: [] };
+      if (error instanceof Error && error.message.includes('Heatmap token does not belong to this project')) {
+        clearHeatmapTokenStorage(app.projectId);
+        serverHeatmapError = 'The stored heatmap token belongs to another project. Generate a fresh token for this project.';
+      } else {
+        serverHeatmapError = error instanceof Error ? error.message : 'Failed to load heatmap data';
+      }
+    } finally {
+      if (!isLatestRequest()) {
+        return;
```

Biome (2.4.15)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)